cardano-cli/test/cardano-cli-golden/files/golden/help/babbage_governance.cli

```diff
@@ -13,6 +14,7 @@ Available options:
 Available commands:
   create-mir-certificate   Create an MIR (Move Instantaneous Rewards)
                            certificate
+  action                   Governance action commands.
```

Comment:
This needs to be fixed.

Reply:
We can actually leave this here as:
1. We support legacy commands
2. We've already broken the api